src/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import { Nav } from './components/Nav';
+import { AddItems } from './pages/Add-items/Add-items';
```

Comment:
Take care with naming component files. [As your teammates](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/16#discussion_r785042507), I suggested reading this article about [best practices with naming conventions](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-pascal-case.md). 👀 😄